packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -446,10 +446,6 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
   };
 
   private _onToggleMenu = (shouldFocusOnContainer: boolean): void => {
-    if (this._splitButtonContainer.current) {
-      this._splitButtonContainer.current.focus();
-    }
```

Comment:
Is this the right fix? It looks like you added it in March to fix some specific issues: https://github.com/OfficeDev/office-ui-fabric-react/pull/4222

Reply:
Like we talked offline, we're getting rid of this and instead we're using the onFocus capture event to ensure that focus is set to the SplitButton Container instead of us manually putting it there.